concordium-sdk/src/main/java/com/concordium/sdk/responses/blocksummary/updates/keys/RootKeysUpdates.java

```diff
@@ -0,0 +1,27 @@
+package com.concordium.sdk.responses.blocksummary.updates.keys;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.util.List;
+
+/**
+ * Root update keys.
+ */
+@Getter
+@ToString
+@EqualsAndHashCode
+public final class RootKeysUpdates {
+    private final List<VerificationKey> verificationKeys;
```

Comment:
Same as in the other place I commented, this would be better as an ArrayList. There's no need for the extra generality as I see it.

Reply:
It will be an `ArrayList` at runtime because that is the default choice of the json deserializing library. I will change it to the concrete type though.